modules/oadp-operator-supported.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-intro.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oadp-operator-supported_{context}"]
+= Support for {oadp-full}
+
+.Supported versions of {oadp-short}
+[width="100%",cols="10%,12%,12%,13%,13%,20%,20%,options="header"]
+|===
+
+|Version
+|{OCP-short} version
```

Comment:
Just an FYI that I don't think {ocp-short} is resolving in the preview. It might work in OpenShift dedicated but [this preview](https://84520--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/oadp-intro#oadp-operator-supported_oadp-api), which is for enterprise, doesn't resolve right. Unless the assembly that contains this module will only ever be in OSD, I think it's probably safer to just write the term out here:
```suggestion
|OCP version
```